### DISCUSSION TITLE
how to assign specific gpus in mmdetection during training?

### DISCUSSION THREAD

**Discussion by GolvenDong** [open-mmlab/mmdetection#8787] [Q&A] [answered]
The documentation says I can use dist_trian.sh to train with mutiple gpus,but how to specify gpus? For example,I want to use gpu:2,3 to train my models.
The official tutorial can only specify the gpu numbers.

**Comment by jbwang1997** [comment, marked as answer]
You can add CUDA_VISIBLE_DEVICES="2,3" in the front of `dist_trainsh`

**Reply by ZwwWayne** [reply]
Note that we can add `CUDA_VISIBLE_DEVICES=2.3` in both dist_train.sh and slurm_train.sh.

**Reply by GolvenDong** [reply]
Okay，I will try it!
thx!
